lib/thumbs/pull_request_worker.rb

```diff
@@ -130,9 +130,27 @@ def run_command_in_docker(command)
       [output, exit_code]
     end
 
+    def otp_installations
+      output, exit_code = `kerl list installations`
+      installations={}
+      output.lines.each do |line|
+        build_name, path = line.split(/\s+/)
+        installations[build_name] = path
+      end
+      installations
+    end
+
     def run_command_in_shell(command)
       shell=thumb_config['shell']||'/bin/bash'
-
+      if thumb_config.key?('otp')
+        build_name=thumb_config['otp']
+        unless otp_installations.key?(build_name)
+          output =  "OTP #{thumb_config.key?('otp')} not present."
```

Comment:
@paegun 
1) [tbd] some thought needed to implement clean logic to install otp versions on demand. another pr. 
2) [done] error now spits out kerl list releases
3) [done] updated README with config option info.